### PR TITLE
add precourse challenges

### PR DESCRIPTION
**Pull request by marcgm97**

### CODE REVIEW

File: staff/marc-guillemas/precourse/pasapalabra-html/index.js [hunk 1]
```diff
@@ -0,0 +1,358 @@
...
+var timeUp;
+var userID;
+var contOfQuestions = 0;
+var random;
+var question;
+var correctAnswers = 0;
+var failedAnswers = 0;
+var obj = {};
+var ranking = [{
+    name: 'Sergi',
```
Comment (luislourencos) [inline staff/marc-guillemas/precourse/pasapalabra-html/index.js:106]: Change the name Sergi  to 'Sérgio Luis'

File: staff/marc-guillemas/precourse/airlines-pro/index.js [hunk 2]
```diff
@@ -0,0 +1,337 @@
...
+  }
+
+  var search = prompt("Indicate if you want to search by upper price, lower price or equal price: \n Type upper, lower or equals");
+
+  while (search != "lower" && search != "upper" && search != "equals") {
+      alert("Choose one of those three options please");
+      search = prompt("Indicate if you want to search by upper price, lower price or equal price: \n Type upper, lower or equals");
+  }
+  var arr = [];
+  for (let i = 0; i < flights.length; i++) {
```
Comment (anderMartin92) [inline staff/marc-guillemas/precourse/airlines-pro/index.js:176]: pls don't use let, we are working in ES5 version bro!

File: staff/marc-guillemas/precourse/pasapalabra-html/index.js (hunk 1 above)
Comment (marcgm97) [reply staff/marc-guillemas/precourse/pasapalabra-html/index.js:106]: okei Sir, your wishes are orders

Comment (luislourencos) [reply staff/marc-guillemas/precourse/pasapalabra-html/index.js:106]: Thanks!!!

File: staff/marc-guillemas/precourse/airlines-pro/index.js (hunk 2 above)
Comment (marcgm97) [reply staff/marc-guillemas/precourse/airlines-pro/index.js:176]: okeii estivii I will correct this in a few minutes

File: staff/marc-guillemas/precourse/pasapalabra-html/index.html [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+                </div>
+                <div id="timer-container">
+                    <h1 id="timer"></h1>
+                </div>
+                <button onclick="quitGame()" id="home-page" type="button" class="btn"><svg class="bi bi-house-fill" width="1em" height="1em" viewBox="0 0 16 16" fill="currentColor" xmlns="http://www.w3.org/2000/svg">
+                    <path fill-rule="evenodd" d="M8 3.293l6 6V13.5a1.5 1.5 0 01-1.5 1.5h-9A1.5 1.5 0 012 13.5V9.293l6-6zm5-.793V6l-2-2V2.5a.5.5 0 01.5-.5h1a.5.5 0 01.5.5z" clip-rule="evenodd"/>
+                    <path fill-rule="evenodd" d="M7.293 1.5a1 1 0 011.414 0l6.647 6.646a.5.5 0 01-.708.708L8 2.207 1.354 8.854a.5.5 0 11-.708-.708L7.293 1.5z" clip-rule="evenodd"/>
+                </svg></button>
+                
+
```
Comment (LLorch1984) [inline staff/marc-guillemas/precourse/pasapalabra-html/index.html:126]: could use more semantic syntax

**Pull request merged**